src/worldfactory.cpp

```diff
@@ -144,20 +143,30 @@ WORLDPTR worldfactory::make_new_world( bool show_prompt )
     // World to return after generating
     WORLDPTR retworld = new WORLD();
     if( show_prompt ) {
-        // Window variables
-        const int iOffsetX = (TERMX > FULL_SCREEN_WIDTH) ? (TERMX - FULL_SCREEN_WIDTH) / 2 : 0;
-        const int iOffsetY = (TERMY > FULL_SCREEN_HEIGHT) ? (TERMY - FULL_SCREEN_HEIGHT) / 2 : 0;
-        // set up window
-        WINDOW *wf_win = newwin(FULL_SCREEN_HEIGHT, FULL_SCREEN_WIDTH, iOffsetY, iOffsetX);
-        WINDOW_PTR wf_winptr( wf_win );
+        
+        UIWindow win(FULL_SCREEN_WIDTH, FULL_SCREEN_HEIGHT, UIWindow::Location::Centered, false);
+        auto tabPanel = std::make_shared<UITabPanel>(true);
+
+        win.m_panel->AddChild(tabPanel);
+        
+        for (auto tabName : tab_strings)
+        {
+            auto thisPanel = std::make_shared<UIPaddingPanel>(false);
```

Comment:
This is another unclear use of a compile time constant boolean. What does the `false` here indicate?